index.d.ts

```diff
@@ -549,6 +543,18 @@ declare module 'discord.js' {
 		width: number;
 	}
 
+	export class MessageMentions {
+		channels: Collection<string, GuildChannel>;
```

Comment:
Technically it's possible to mention VoiceChannels with `<#ID>`, but I doubt that has any use case since that would require the user to manually copy the id and make the mention themselves.